src/controllers/sdex/index.ts

```diff
@@ -29,12 +37,38 @@ router.get(
   }),
 );
 
-router.get('/user_pool_positions', async (req, res) => {
-  res.status(200).json({
-    data: {
-      user_pool_positions: [],
-    },
-  });
-});
+router.get(
+  '/user_pool_positions',
+  asyncRoute(async (req, res) => {
+    const { userId } = req.query;
+
+    if (!userId || typeof userId !== 'string') {
+      return res.status(400).json({ error: 'User ID is required' });
+    }
```

Comment:
Use validation instead:
```
const { user } = validate(Joi.object({
 user: Joi.string().required(),
}), req.query);
```

Also, use `user` instead of `userId` in query params to make it compatible with bob indexer